usaspending_api/disaster/tests/fixtures/disaster_account_data.py

```diff
@@ -0,0 +1,290 @@
+import pytest
+
+from model_mommy import mommy
+
+
+@pytest.fixture
+def disaster_account_data():
+    ta1 = mommy.make("references.ToptierAgency", toptier_code="007")
+    ta2 = mommy.make("references.ToptierAgency", toptier_code="008")
+    ta3 = mommy.make("references.ToptierAgency", toptier_code="009")
+    ta4 = mommy.make("references.ToptierAgency", toptier_code="010")
+
+    sub1 = mommy.make(
+        "submissions.SubmissionAttributes",
+        reporting_fiscal_year="2022",
```

Comment:
I'm surprised strings work better for these fields

Reply:
I can revert this back to integers. Was left as part of my attempts to get tests working for Travis.

Reply:
Doesn't matter to me if it works